### DISCUSSION TITLE
Version v0.1.11rc3 Release Today!

### DISCUSSION THREAD

**Discussion by ver217** [hpcaitech/ColossalAI#1932] [Announcements]
### Release 
- [release] update version (#1931) by [ver217](https://api.github.com/users/ver217)

### Tutorial 
- [tutorial] polish README and OPT files (#1930) by [binmakeswell](https://api.github.com/users/binmakeswell)
- [tutorial] add synthetic dataset for opt (#1924) by [ver217](https://api.github.com/users/ver217)
- [tutorial] updated hybrid parallel readme (#1928) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] added synthetic data for sequence parallel (#1927) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] removed huggingface model warning (#1925) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- Hotfix/tutorial readme index (#1922) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] modify hands-on of auto activation checkpoint (#1920) by [Boyuan Yao](https://api.github.com/users/Cypher30)
- [tutorial] added synthetic data for hybrid parallel (#1921) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] added synthetic data for hybrid parallel (#1919) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] added synthetic dataset for auto parallel demo (#1918) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] updated auto parallel demo with latest data path (#1917) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] added data script and updated readme (#1916) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] add cifar10 for diffusion (#1907) by [binmakeswell](https://api.github.com/users/binmakeswell)
- [tutorial] removed duplicated tutorials (#1904) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [tutorial] edited hands-on practices (#1899) by [BoxiangW](https://api.github.com/users/BoxiangW)

### Example 
- [example] update auto_parallel img path (#1910) by [binmakeswell](https://api.github.com/users/binmakeswell)
- [example] add cifar10 dadaset for diffusion (#1902) by [Fazzie-Maqianli](https://api.github.com/users/Fazziekey)
- [example] migrate diffusion and auto_parallel hands-on (#1871) by [binmakeswell](https://api.github.com/users/binmakeswell)
- [example] initialize tutorial (#1865) by [binmakeswell](https://api.github.com/users/binmakeswell)
- Merge pull request #1842 from feifeibear/jiarui/polish by [Fazzie-Maqianli](https://api.github.com/users/Fazziekey)
- [example] polish diffusion readme by [jiaruifang](https://api.github.com/users/feifeibear)

### Sc 
- [SC] add GPT example for auto checkpoint (#1889) by [Boyuan Yao](https://api.github.com/users/Cypher30)
- [sc] add examples for auto checkpoint. (#1880) by [Super Daniel](https://api.github.com/users/super-dainiu)

### Nfc 
- [NFC] polish colossalai/amp/naive_amp/__init__.py code style (#1905) by [Junming Wu](https://api.github.com/users/jason524w)
- [NFC] remove redundant dependency (#1869) by [binmakeswell](https://api.github.com/users/binmakeswell)
- [NFC] polish .github/workflows/scripts/build_colossalai_wheel.py code style (#1856) by [yuxuan-lou](https://api.github.com/users/yuxuan-lou)
- [NFC] polish .github/workflows/scripts/generate_release_draft.py code style (#1855) by [Ofey Chan](https://api.github.com/users/ofey404)
- [NFC] polish workflows code style (#1854) by [Kai Wang (Victor Kai)](https://api.github.com/users/kaiwang960112)
- [NFC] polish colossalai/amp/apex_amp/__init__.py code style (#1853) by [LuGY](https://api.github.com/users/Gy-Lu)
- [NFC] polish .readthedocs.yaml code style (#1852) by [nuszzh](https://api.github.com/users/nuszzh)
- [NFC] polish <.github/workflows/release_nightly.yml> code style (#1851) by [RichardoLuo](https://api.github.com/users/RichardoLuo)
- [NFC] polish amp.naive_amp.grad_scaler  code style by [zbian](https://api.github.com/users/kurisusnowdeng)
- [NFC] polish colossalai/auto_parallel/tensor_shard/deprecated/op_handler/operator_handler.py code style (#1845) by [HELSON](https://api.github.com/users/1SAA)
- [NFC] polish ./colossalai/amp/torch_amp/__init__.py code style (#1836) by [Genghan Zhang](https://api.github.com/users/zhang677)
- [NFC] polish .github/workflows/build.yml code style (#1837) by [xyupeng](https://api.github.com/users/xyupeng)
- [NFC] polish colossalai/auto_parallel/tensor_shard/deprecated/op_handler/conv_handler.py code style (#1829) by [Sze-qq](https://api.github.com/users/Sze-qq)
- [NFC] polish colossalai/amp/torch_amp/_grad_scaler.py code style (#1823) by [Ziyue Jiang](https://api.github.com/users/Wesley-Jzy)
- [NFC] polish .github/workflows/release_docker.yml code style by [Maruyama_Aya](https://api.github.com/users/MaruyamaAya)
- [NFC] polish .github/workflows/submodule.yml code style (#1822) by [shenggan](https://api.github.com/users/Shenggan)
- [NFC] polish .github/workflows/draft_github_release_post.yml code style (#1820) by [Arsmart1](https://api.github.com/users/Arsmart1)
- [NFC] polish colossalai/amp/naive_amp/_fp16_optimizer.py code style (#1819) by [Fazzie-Maqianli](https://api.github.com/users/Fazziekey)
- [NFC] polish colossalai/amp/naive_amp/_utils.py code style (#1816) by [CsRic](https://api.github.com/users/CsRic)
- [NFC] polish .github/workflows/build_gpu_8.yml code style (#1813) by [Zangwei Zheng](https://api.github.com/users/zhengzangw)
- [NFC] polish MANIFEST.in code style (#1814) by [Zirui Zhu](https://api.github.com/users/ziruizhu)
- [NFC] polish strategies_constructor.py code style (#1806) by [binmakeswell](https://api.github.com/users/binmakeswell)

### Doc 
- [doc] add news (#1901) by [binmakeswell](https://api.github.com/users/binmakeswell)

### Zero 
- [zero] migrate zero1&2  (#1878) by [HELSON](https://api.github.com/users/1SAA)

### Autoparallel 
- [autoparallel] user-friendly API for CheckpointSolver. (#1879) by [Super Daniel](https://api.github.com/users/super-dainiu)
- [autoparallel] fix linear logical convert issue (#1857) by [YuliangLiu0306](https://api.github.com/users/YuliangLiu0306)

### Fx 
- [fx] metainfo_trace as an API. (#1873) by [Super Daniel](https://api.github.com/users/super-dainiu)

### Hotfix 
- [hotfix] pass test_complete_workflow (#1877) by [Jiarui Fang](https://api.github.com/users/feifeibear)

### Inference 
- [inference] overlap comm and compute in Linear1D_Row when stream_chunk_num > 1 (#1876) by [Jiarui Fang](https://api.github.com/users/feifeibear)
- [inference] streaming Linear 1D Row inference (#1874) by [Jiarui Fang](https://api.github.com/users/feifeibear)

### Amp 
- [amp] add torch amp test (#1860) by [xcnick](https://api.github.com/users/xcnick)

### Diffusion 
- [diffusion] fix package conflicts (#1875) by [HELSON](https://api.github.com/users/1SAA)

### Utils 
- [utils] fixed lazy init context (#1867) by [Frank Lee](https://api.github.com/users/FrankLeeeee)
- [utils] remove lazy_memory_allocate from ColoInitContext (#1844) by [Jiarui Fang](https://api.github.com/users/feifeibear)

**Full Changelog**: https://github.com/hpcaitech/ColossalAI/compare/v0.1.11rc3...v0.1.11rc2

<hr /><em>This discussion was created from the release <a href='https://github.com/hpcaitech/ColossalAI/releases/tag/v0.1.11rc3'>Version v0.1.11rc3 Release Today!</a>.</em>

**Comment by mosesojero** [comment]
Update

**Comment by JoeTorrey** [comment]
New to this but would like to know and learn more about it very interested

**Comment by noradeebabe** [comment]
Good

**Reply by noradeebabe** [reply]
Ok
